src/idle_mapping.ts

```diff
@@ -70,3 +71,25 @@ export function handleVoteCast(event: VoteCast): void {
   vote.organization = org.id;
   vote.save();
 }
+
+export function handleVoteCastAlpha(event: VoteCastAlpha): void {
```

Comment:
Why do we need this new method? It is exact copy of handleVoteCast. Why can't you use that method?

Reply:
The ABI's are different:
- VoteCast(indexed address,uint256,uint8,uint256,string)
- VoteCast(address,uint256,bool,uint256)

Reply:
I can try use 
``` export function handleVoteCastAlpha(event: VoteCast | VoteCastAlpha)``` 
but i think maybe the code will be a little messed up

Reply:
or extract most of the code into a private function and call that from both handleVoteast and handleVoteastAlpha?